test/models/patient_test.rb

```diff
@@ -1985,6 +1985,44 @@ def verify_patient_status(patient, status)
     assert patient.valid?
   end
 
+  test 'validates follow_up_reason is a valid reason in api context' do
+    patient = valid_patient
+
+    patient.follow_up_reason = 'Deceased'
+    assert patient.valid?(:api)
+
+    patient.follow_up_reason = 'Duplicate'
+    assert patient.valid?(:api)
+
+    patient.follow_up_reason = 'High-Risk'
+    assert patient.valid?(:api)
+
+    patient.follow_up_reason = 'Other'
+    assert patient.valid?(:api)
```

Comment:
In this case I think it is fine to test just one valid value, since the rest of the valid values are essentially an equivalent case.